packages/components/package.json

```diff
@@ -30,7 +30,7 @@
     "start": "ember serve",
     "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each",
+    "test:ember-compatibility": "ember try:one",
```

Comment:
last commit to give error 

```
Module not found: Error: Can't resolve '@hashicorp/design-system-tokens/dist/docs/products/tokens.json' in '/tmp/broccoli-1986oNh5lrWv9H3o/cache-244-webpack_bundler_ember_auto_import_webpack'
resolve '@hashicorp/design-system-tokens/dist/docs/products/tokens.json' in '/tmp/broccoli-1986oNh5lrWv9H3o/cache-244-webpack_bundler_ember_auto_import_webpack'
```

https://github.com/hashicorp/design-system/runs/5547634475?check_suite_focus=true#step:5:69